app/assets/stylesheets/components/_locator-map.scss

```diff
@@ -0,0 +1,29 @@
+.location-search-types {
+  @include margin($site-margins null);
+  border: solid $color-gray-light;
+  border-width: 1px 0;
+
+  .usa-media_block {
+    @include margin($site-margins null);
+  }
+
+  p {
+    margin: 0;
+  }
+}
+
+.locator-map {
+  @include margin($site-margins null);
+  background-color: $color-gray-lightest;
+  height: 40rem;
+}
+
+.map-marker-bubble {
+  max-width: 24rem;
+  text-align: center;
+
+  b {
+    display: block;
+    margin-bottom: 0.25rem;
+  }
+}
```

Comment:
I don't see the animating location dot anymore. Not to your taste?

Reply:
@jamesatheyDDS Ah, good catch. Sorry about that!